CMakeLists.txt

```diff
@@ -8,10 +8,10 @@ project(
 #
 # options
 #
-#
-set(CMAKE_INSTALL_PREFIX "/opt/poky/rzn1/sysroots/x86_64-pokysdk-linux/usr")
+# <INSERT_CMAKE_INSTALL_PREFIX_HERE>
 #set(CMAKE_INSTALL_PREFIX "/opt/poky/rzn1/sysroots/x86_64-pokysdk-linux/usr")
 
+
```

Comment:
Why the extra empty line? :)

Reply:
Because I'm half asleep atm and not fully thinking yet ha